File: extract_scores.py
import json

try:
    with open('lighthouse-report.json', 'r', encoding='utf-8') as f:
        data = json.load(f)
        
    categories = data.get('categories', {})
    
    perf = categories.get('performance', {}).get('score', 0) * 100
    acc = categories.get('accessibility', {}).get('score', 0) * 100
    bp = categories.get('best-practices', {}).get('score', 0) * 100
    seo = categories.get('seo', {}).get('score', 0) * 100
    
    print(f"Performance: {perf}")
    print(f"Accessibility: {acc}")
    print(f"Best Practices: {bp}")
    print(f"SEO: {seo}")

except Exception as e:
    print(f"Error: {e}")

File: extract_scores.js
const fs = require('fs');

try {
    const data = fs.readFileSync('lighthouse-report.json', 'utf8');
    const report = JSON.parse(data);
    const categories = report.categories;

    console.log('Performance:', categories.performance.score * 100);
    console.log('Accessibility:', categories.accessibility.score * 100);
    console.log('Best Practices:', categories['best-practices'].score * 100);
    console.log('SEO:', categories.seo.score * 100);

} catch (err) {
    console.error('Error parsing report:', err);
}
